### ISSUE TITLE
Theme is not set properly 

### ISSUE THREAD

**Issue by gkeegan** [afollestad/mnml#55]
The app refuses to exit dark mode. This may be due to having omnirom theming on my device (even though that is set to light mode).

![Screenshot](https://i.imgur.com/dZOJlO7.png)

**Comment by afollestad** [comment]
Hmmm. Your phone might be telling MNML that night mode is enabled globally on your device, in which case the resources are overriding the manual setting which should probably be hidden.

**Comment by gkeegan** [comment]
Yeah I figured it was my phones themes. The theming doesn't apply to any other non-system apps, so I'm confused why it did it with MNML.

**Comment by afollestad** [comment]
MNML specifies night theme overrides that Android will automatically use if the device is in night mode, this is a recent change for Android Q dark mode support.

**Comment by gkeegan** [comment]
Oh I see. That makes sense then.
